Gemfile

```
@@ -45,4 +45,4 @@ group :test do
   gem 'rails-controller-testing' # Rails 4 fallback
 end
 
-gem 'sparkpost_rails'
+gem 'sparkpost_rails'
```

I wonder if applying this change would fix the missing newline...

```suggestion
gem 'sparkpost_rails'
```